wpimath/src/main/native/include/frc/controller/SimpleMotorFeedforward.h

```diff
@@ -140,19 +170,15 @@ class SimpleMotorFeedforward {
       //
       // Substitute these into the feedforward equation.
       //
-      //   uₖ = B_d⁺(rₖ₊₁ − A_d rₖ)
-      //   uₖ = (1/kᵥ)⁺(rₖ₊₁ − (0) rₖ)
-      //   uₖ = kᵥrₖ₊₁
+      //   uₖ = B_d⁺(xₖ₊₁ − A_d xₖ) + kₛ sgn(x)
+      //   uₖ = (1/kᵥ)⁺(xₖ₊₁ − (0) xₖ) + kₛ sgn(x)
+      //   uₖ = kᵥxₖ₊₁  + kₛ sgn(x)
```

Comment:
The order of terms doesn't match the code.

Reply:
So like this ? 

uₖ = kₛ sgn(x) + B_d⁺(xₖ₊₁ − A_d xₖ)

Reply:
Yea.

Reply:
Would you like this for every line or just at this point in the code?

Reply:
Rearranging it at the end is fine.